R/tcplVarMat.R

```diff
@@ -31,15 +31,16 @@
 #' 
 #' \code{tcplVarMat} produces matrices of combined sc-mc output. For the ac50
 #' and acc matrices specifically, values are inserted in place to show complete
-#' views of what was tested and what the results were. ac50 and acc values are
-#' set to:
-#' - 1e6 when the chemical is tested but negative in mc
-#' - 1e7 when the chemical is not tested in mc but was screened in sc with a
+#' views of what was tested and what the results were. ac50 and acc values are:
+#' \itemize{
+#'  \item set to 1e6 when the chemical is tested but negative in mc
+#'  \item set to 1e7 when the chemical is not tested in mc but was screened in sc with a
```

Comment:
set to 1e7 when the chemical is not tested in mc but was screened in sc with a positive hitcall for the same aeid. In _verbose sheets, these are indicated as "SC pos, No MC"